components/raftstore/src/store/config.rs

```diff
@@ -29,8 +29,9 @@ with_prefix!(prefix_store "store-");
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
-    // true for high reliability, prevent data loss when power failure.
-    pub sync_log: bool,
+    // delay time of raft db sync (us).
+    #[config(skip)]
```

Comment:
Why skip?

Reply:
Because it can not be dynamically updated.

Reply:
🤔Maybe it can. If users enable fsync control(set delay-sync-us not 0), it can be dynamically updated to other values except 0. If users want to change it to 0 dynamically, we should do more work to make it correct.
For example, https://github.com/gengliqi/tikv/blob/fsync-control-4.0/components/raftstore/src/store/fsm/store.rs#L605-L614, we can consider it is synced when the write batch is empty and fsync control is disabled.